src/App.js

```diff
@@ -1,12 +1,14 @@
 import HomeHeader from "./components/header/HomeHeader.jsx";
 import HomeHeaderMobile from "./components/headerMobile/HomeHeaderMobile.jsx";
+import VoucherHeader from "./components/voucherHeader/VoucherHeader.jsx";
 
 import "./styles/global.css";
 function App() {
   return (
     <div className="App">
       <HomeHeader />
       <HomeHeaderMobile />
+      <VoucherHeader />
```

Comment:
There’s  a spacing  here because the margin and padding has not been reset, its necessary to reset all spacings, you can learn about  [here](https://css-tricks.com/reset-all-margins-padding/) 

<img width="816" alt="Screen Shot 2022-02-08 at 13 18 07" src="https://user-images.githubusercontent.com/38358239/153029590-3c121907-95ce-479a-a132-b0fe6c68e87e.png">

Reply:
fixed rai, i read about 👀 

<img width="1678" alt="Captura de Tela 2022-02-08 às 21 17 02" src="https://user-images.githubusercontent.com/42494406/153098103-4680e4ce-bcd6-405b-9948-eb4533588ca6.png">